lib/scope.js

```diff
@@ -1086,15 +928,57 @@ module.exports = {
           }
 
           if ( rows_match ) {
-            matches.push({ field: field, table: table_row });
+            // Collect all the matches for each individual field - we
+            // may need to de-duplicate them.
+            if ( matches_by_field[ field.selector ] ) {  // selector should be unique per field
+              matches_by_field[ field.selector ].push({ field: field, table: table_row });
+            } else {
+              matches_by_field[ field.selector ] = [{ field: field, table: table_row }];
+            }
+
             continue;
           }
 
         }  // ends for each table row
       }  // ends for each field row possibility
     }  // ends for each field
 
-    return matches;
+    // Some fields will have multiple matches from the story table.
```

Comment:
The only reason we would care about multiple rows is if there is a possible proxy var conflict.

Reply:
Comment added.